CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Cyclone
+
+Welcome to Cyclone! This guide will provide information on how to contribute to Cyclone. It covers how to build Cyclone, how to commit and what conventions and practices contributors should follow.
+
+If you have any problems about Cyclone, welcome to join [Cyclone Slack](https://caicloud-cyclone.slack.com) to discuss.
+
+## Get Start
+
+### Fork Repository
+
+Fork Cyclone repository to your own Github account, and then:
+```
+$ mkdir -p $GOPATH/src/github.com/caicloud
+$ go get github.com/caicloud/cyclone
+
+# Avoid push to caicloud/cyclone
+$ git config push.default nothing
+
+$ git remove add $USER git@github.com:$USER/cyclone.git
+$ git fetch $USER
+```
+
+### Project Structure
+
+```
+.
+...
+├── Makefile  # Make file to build and package
+├── build     # Contains files to build images, e.g. Dockerfiles
+├── cmd       # Entrypoints (main.go) of components
+├── manifests # Kubernetes manifests for deployment and examples
+├── pkg       # Source code for Cyclone server, workflow engine
+├── tools     # Tools used by Cyclone, like Kubernetes client generator
+└── web       # Web UI source code for Cyclone
+```
+
+### Build & Start
+
+About how to build and start Cyclone, please refer to [Build & Run](./docs/build-guide.md).
+
+## Contribute Workflow
```

Comment:
Refer to [github-workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)